WordPress/src/jetpack/java/org/wordpress/android/ui/accounts/login/components/RepeatingColumn.kt

```diff
@@ -24,7 +25,7 @@ fun RepeatingColumn(
 ) {
     Layout(
             content = {
-                Column(modifier = modifier) {
+                Column(modifier.clearAndSetSemantics {}) {
```

Comment:
I was thinking it might be better to leave this as is, and instead add `clearAndSetSemantics` to the `LoopingTextWithBackground`, since that makes this component more reusable. Wdyt?

Reply:
Makes sense to me Matthew 👍 , the fact that it remained in `RepeatingColumn` is because I was trying other options that made sense to be placed here (`semantics(mergeDescendants = true) {...})`) here but not anymore after using `clearAndSetSemantics {}` 😅 

Thank you 🙇